[ACM-5757]: Adding info about multiclusterengine commands

lahinson opened the pull request:
Related Jira: https://issues.redhat.com/browse/ACM-5757

This PR adds information to instances of the following commands in the hosted control plane docs:
- oc patch mce multiclusterengine
- oc get mce multiclusterengine

The purpose of the additional information is to let users know what the default MultiClusterEngine resource instance name is and how they can find out what the instance name is from their hosted cluster.

lahinson committed:
[ACM-5757]: Adding info about multiclusterengine commands

lahinson committed:
adding formatting

rokej reviewed (approved):
Thank you!

dockerymick commented on clusters/hosted_control_planes/configure_hosted_aws.adoc:
```diff
@@ -377,14 +377,16 @@ The hosted control planes feature is disabled by default. Enabling the feature a
-oc get mce multiclusterengine -o yaml
+oc get mce multiclusterengine -o yaml <1>
 ----
+<1> The default `MultiClusterEngine` resource instance name is `multiclusterengine`, but you can get the `MultiClusterEngine` name from your cluster by running the following command: `$ oc get mce`.
```
Changes look good in this file, I recommend that you remove the link to the Additional resources section from the table of contents. In an earlier doc meeting, we agreed that the link to the section was unnecessary.

dockerymick reviewed (approved):
One comment, very small change. I can also create an issue for myself to remove the `<<additional-resources-x,Additional resources>>` syntax throughout the docs. Let me know your thoughts

lahinson committed:
removing additional resources anchor link

lahinson commented:
@dockerymick Thanks so much for your review! I removed the Additional resources anchor link -- good catch! I think I'd hold off on creating an issue to remove those links throughout the docs. I have a big PR in progress to restructure some of the bare metal docs for hosted control planes, and I removed the "Additional resources" links from those files as part of that work. Maybe after that PR is merged, if we're still seeing a lot of "Additional resources" links, we can do a doc-wide scrub.

dockerymick commented:
@lahinson thanks for the heads up and suggestion! I agree with that, I'll hold off on creating the issue
